### PR TITLE
Add serviceRunner for container service scaling

### PR DESCRIPTION
**Pull request by fishnix**
```json
		"details": {
			"runner": "serviceRunner",
			"service_cluster": "my_cluster",
                         "service_name": "my_service",
                         "service_action": "scale",
                         "desired_count": "2"
		},
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,31 @@ An instance runner job executes an action on an instance.  Currently supported a
 }
 ```
 
+### service
```
Comment (tenyo) [inline README.md:79]: maybe call it `container service`?

File: README.md [hunk 2]
```diff
@@ -76,6 +76,31 @@ An instance runner job executes an action on an instance.  Currently supported a
...
+
+A service runner job executes an action on an service.  Currently supported action is `scale`.
```
Comment (tenyo) [inline README.md:81]: 👀 `on a service`

File: jobs/servicerunner.go [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+		if r.Token != "" {
+			log.Debugf("setting token header %s for %s/%s", r.AuthHeader, s.Cluster, s.Name)
+			if r.Encrypt {
+				e, err := bcrypt.GenerateFromPassword([]byte(r.Token), 6)
+				if err != nil {
+					return "", NewRunnerError(ErrExecFailure, "unable to hash token", err)
+				}
+
+				log.Debugf("token is encrypted, setting header to %s", string(e))
```
Comment (tenyo) [inline jobs/servicerunner.go:151]: do we need to log the actual token here?

File: README.md (hunk 1 above)
Comment (fishnix) [reply README.md:79]: i welcome the future "i told you so" but i think this is fine for now 😆

**Comment by damnski** [comment]
:+1:

**Pull request merged**